konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/bbox/filter/BoundingBoxFilterStepRunner.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  ******************************************************************************
+ *  * Copyright (c) 2020 Konduit K.K.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+package ai.konduit.serving.pipeline.impl.step.bbox.filter;
+
+import ai.konduit.serving.pipeline.api.context.Context;
+import ai.konduit.serving.pipeline.api.data.BoundingBox;
+import ai.konduit.serving.pipeline.api.data.Data;
+import ai.konduit.serving.pipeline.api.data.ValueType;
+import ai.konduit.serving.pipeline.api.step.PipelineStep;
+import ai.konduit.serving.pipeline.api.step.PipelineStepRunner;
+import lombok.AllArgsConstructor;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+public class BoundingBoxFilterStepRunner implements PipelineStepRunner {
+
+    protected final BoundingBoxFilterStep step;
```

Comment:
Let's replace the `@AllArgsConstructor` with manually created contructor (i.e., `public BoundingBoxFilterStepRunner(BoundingBoxFilterStep step){ ...}` and validate that the classesToKeep setting is not null and has length > 0.
Otherwise if a user forgets to set the classes, we could filter everything and they won't know why their pipeline doesn't work.